content/en/blog/2023/contributing-to-otel/index.md

```diff
@@ -0,0 +1,191 @@
+  demonstrate OpenTelemetry instrumentation and Observability.
+- Build a base for vendors, tooling authors, and others to extend and
+  demonstrate their OpenTelemetry integrations.
+- Create a living example for OpenTelemetry contributors to use for testing new
+  versions of the API, SDK, and other components or enhancements.
+
+The OTel Demo [just celebrated its first birthday](/blog/2023/demo-birthday/),
+and it's come a long way since its first release. As OTel evolves, the OTel Demo
+must also evolve alongside it, to ensure that it is utilizing the latest and
+greatest versions of things like the [OTel Collector](/docs/collector/) and
+language-specific instrumentation. In short, there's always work to be done!
+
+You can check out an example of one of my contributions
+[here](https://github.com/open-telemetry/opentelemetry-demo/pull/432).
+
+Learn more about how to contribute to the OTel Demo
+[here](https://github.com/open-telemetry/opentelemetry-demo/blob/main/CONTRIBUTING.md).
+
+### Join a Special Interest Group
+
+Feeling a little more adventurous? Then why not join one of the other OTel
+[special interest groups](https://en.wikipedia.org/wiki/Special_interest_group)
+(SIGs)? By joining a SIG, you can contribute to things like (but not limited to)
+SDK development, auto-instrumentation of libraries, the
+[OTel Collector](/docs/collector/), and the
+[OTel Operator](/docs/kubernetes/operator/). You don't even need to be a regular
+contributor. Sometimes, contributions come out of necessity. Like maybe you
+found a bug in the Collector. Why not take a stab at fixing it? Or maybe there's
+a feature that you and your team desperately need. Again, why not take a stab at
+implementing it?
+
+Now, before you go off writing code, you should make sure that you:
+
+1. Raise an issue in the
+   [appropriate GitHub repository](https://github.com/open-telemetry)
+2. Check with folks on
+   [Slack](https://communityinviter.com/apps/cloud-native/cncf) to see if this
+   feature is needed/wanted
+3. Pop into one of the [SIG calls](https://shorturl.at/beJ09) and share your
+   specific interests and needs
+
+I'm under no illusion that any contributing code to OTel is is a trivial task.
+Far from it. If you do end up contributing code, one way to alleviate the
+stress/learning curve of doing it solo would be to pair with a willing OTel
+community member to implement these types of code changes. I've found folks
+within OTel to be nothing but helpful and welcoming.
+
+You can check out an example of one of my contributions
+[here](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/531).
+
+Learn more about the OTel SIGs [here](/community/#special-interest-groups).
+
+### Final thoughts
+
+As we've seen, there are many ways to contribute to OpenTelemetry, whether it's
+through docs contributions, blogging, joining the End User Working Group,
+contributing to the OTel Demo, or joining a SIG. Anything helps!
```

Comment:
It's maybe worth to emphasis that there are also contributions like "giving talks about OpenTelemetry", "telling your friends and family (and co-workers) how awesome OpenTelemetry is", so also things outside the repositories and official infrastructure (slack, zooms, cncf thingies) is highly helpful